nupic/research/frameworks/vernon/mixins/oml.py

```diff
@@ -156,14 +158,16 @@ def run_meta_testing_phase(self, num_classes_learned):
                 model=self.model.module,
                 loader=self.test_train_loader,
                 optimizer=optim,
-                device=self.device
+                device=self.device,
+                criterion=self._loss_function,
             )
 
             # meta-testing testing
             results = evaluate_model(
                 model=self.model.module,
                 loader=self.test_test_loader,
-                device=self.device
+                device=self.device,
+                criterion=self._loss_function,
```

Comment:
Good catch!

Reply:
Thanks, this was an interesting one hunt down.